src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsUapHiVi.java

```diff
@@ -185,12 +194,14 @@ public void shouldDisplayResolvedStateOnNextPageView(
       double impactStateAspectRatio, double resolvedStateAspectRatio
   ) {
     HiViUap hiViUap = new HiViUap(driver, slotName);
+    String CURRENT_URL = driver.getCurrentUrl();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)